.travis.yml

```diff
@@ -9,12 +9,17 @@ rvm:
   - '2.4.0'
 
 matrix:
+  allow_failures:
+    - rvm: '2.0'
```

Comment:
I wouldn't prefer to allow failures on ruby-2.0 just because of yard-junk. Especially, there are quite a few features missing in ruby-2.0 (like `array.sort_by(n)`) which shouldn't overlooked by passing Travis CI builds. 

P.S - We are in the process of dropping support for ruby-2.0 as per issue #336.But until we support drop for ruby-2.0, let's keep it the correct form?